### PR TITLE
Updated site for 2018

### LINKED ISSUE #32: Update information for the new year
Includes:
- Recruitment information
- Updated events info
- Updated sponsorship information

### PR DESCRIPTION
**Pull request by ValRat**
Closes #32

### CODE REVIEW

File: roster.html [hunk 1]
```diff
@@ -268,6 +268,41 @@ <h4>Martin Freeman</h4>
...
+                <h2>Our Robots</h2>
+                <p>
+                    These are our significant others on Saturdays.
+                </p>
+            </header>
+            <ul class="list-inline team-member">
+                <li>
+                    <img src="images/roster/avalanche.jpg" alt="Avalanche" />
+                    <h4>Avalanche</h4>
+                    <p>Deceased Grandfather</p>
```
Comment (SimonJinaphant) [inline roster.html:285]: That's one morbid description 😲

**Review by SimonJinaphant — APPROVED** [review]
Also how can this be a new year commit if you don't change the footer 2017 to 2018 😉

File: roster.html (hunk 1 above)
Comment (ValRat) [reply roster.html:285]: May he rest in peace.

**Comment by ValRat** [comment]
Ya got me

File: join.html [hunk 2]
```diff
@@ -61,24 +61,31 @@ <h2>UBC Snowbots</h2>
...
                 </blockquote>
                 <p>
                     Check our facebook page for more information regarding upcoming events.
                 </p>
             </section>
             <header class="major">
-                <h3>Recruitment Process</h3>
-                <h4>NOTE: Recruitment for the Mechanical Team is closed. <br/> It will be reopened in January, so stay tuned!</h4>
+                <h2>Recruitment</h2>
+                <h4>Recruitment for Software is open, please check out the <a href="challenge.html">software challenge</a></h4>
```
Comment (Dulluhan) [inline join.html:77]: [note] href is not actually visible on live due to font.

File: index.html [hunk 3]
```diff
@@ -69,25 +69,22 @@ <h2>We are Snowbots</h2>
...
 			</header>
 
 			<ul class="features">
 				<li>
 					<h3 class="icon style1 fa-wrench">Mechanical <br/><span class="sub_h3">overview</span></h3>
 					<p>
-						Our robot uses a LIDAR laser scanner for detecting physical objects and a camera for detecting coloured boundaries painted
-						on the grounds.
+						The Mechanical team designs and constructs the drivetrain and chassis of the robot 
+						and are also responsible for the power systems.
```
Comment (Dulluhan) [inline index.html:80]: [note] either plural robots or singular power system

**Pull request merged**